### DISCUSSION TITLE
Command Post not scanning Avid for Speed Editor setup

### DISCUSSION THREAD

**Discussion by BenEditor** [CommandPost/CommandPost#3252] [Technical Support]
I have just purchased a speed editor with the hope of running it with Avid but when I try to enter commands it can not succesfully open and scan Avid and os is just left loading....  please somebody help

**Comment by latenitefilms** [comment]
Make sure you have these two Search Console items ticked in General Preferences:

<img width="1000" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/88d3139c-21e2-4b11-b2ea-e15af021b1a4">

**Comment by BenEditor** [comment]
Hi Chris,Thanks for the help but those are ticked already.Any other ideas gratefully appreciated.
Best regardsBen Elliott
Mob:07807029494

On Friday, August 11, 2023, 10:53 pm, Chris Hocking ***@***.***> wrote:




Make sure you have these two Search Console items ticked in General Preferences:

**Reply by latenitefilms** [reply]
Ummm... try restarting your Mac, open Avid Media Composer and open a project, THEN try launching CommandPost to see if the menu items are populated. If they still aren't, are there any errors in CommandPost's Debug Console.

We have quite a few Avid Media Composer users, so it SHOULD work fine.
